frontend/packages/integration-tests-cypress/tests/monitoring/monitoring.spec.ts

```diff
@@ -0,0 +1,153 @@
+import { checkErrors, testName } from '../../support';
+import { submitButton, errorMessage } from '../../views/form';
+import { listPage } from '../../views/list-page';
+import { detailsPage } from '../../views/details-page';
+import { modal } from '../../views/modal';
+
+const shouldBeWatchdogAlertDetailsPage = () => {
+  detailsPage.titleShouldBe('Watchdog');
+  detailsPage.sectionHeaderShouldExist('Alert Details');
+  detailsPage.labelShouldExist('alertname=Watchdog');
+};
+
+const shouldBeWatchdogAlertRulesPage = () => {
+  detailsPage.titleShouldBe('Watchdog');
+  detailsPage.sectionHeaderShouldExist('Alerting Rule Details');
+  detailsPage.sectionHeaderShouldExist('Active Alerts');
+};
+
+const shouldBeWatchdogSilencePage = () => {
+  detailsPage.titleShouldBe('Watchdog');
+  detailsPage.sectionHeaderShouldExist('Silence Details');
+  detailsPage.labelShouldExist('alertname=Watchdog');
+};
+
+describe('Monitoring: Alerts', () => {
+  before(() => {
+    cy.login();
+    cy.createProject(testName);
+  });
+
+  afterEach(() => {
+    checkErrors();
+  });
+
+  after(() => {
+    cy.deleteProject(testName);
+    cy.logout();
+  });
+
+  // each it() should be its own atomic test (run independently of other tests), per best practices.
```

Comment:
I think we need to call this out in the README instead (if we haven't) and remove this comment.

Reply:
> I had converted it all, but @kyoto recently added await testSilenceTimeInputs(); to it wink

>> Can we migrate that over as well? I don't think we should have two copies of these tests running.

Yes, that has already been added.

Reply:
fixed, updated README and the `testSilenceTimeInputs` was already ported to Cypress.